.github/workflows/num-rules.yaml

```diff
@@ -0,0 +1,22 @@
+name: num-rules
```

Comment:
Do you know why this appears as "added?" I think this file was already in the repository. I guess maybe it got formatted?